copyparty/web/browser.js

```diff
@@ -7034,6 +7191,8 @@ var ahotkeys = function (e) {
 	if (aet && aet != 'a' && aet != 'tr' && aet != 'td' && aet != 'div' && aet != 'pre')
 		return;
 
+	if (handle_qsearch_key(e)) return;
```

Comment:
The placement of this is very tricky, but I think this should be right? Please verify